src/components/interfaces/HMI_API.xml

```diff
@@ -1569,6 +1649,32 @@
   </param>
 </struct>
 
+<struct name="VideoStreamingFormat">
+  <description>Video streaming formats and their specifications.</description>
+  <param name="protocol" type="Common.VideoStreamingProtocol" mandatory="true">
+    <description>Protocol type, see VideoStreamingProtocol</description>
+  </param>
+  <param name="codec" type="Common.VideoStreamingCodec" mandatory="true">
+    <description>Codec type, see VideoStreamingCodec</description>
+  </param>
+</struct>
+
+<struct name="VideoConfig">
+  <description>Configuration of a video stream.</description>
+  <param name="protocol" type="Common.VideoStreamingProtocol" mandatory="false">
+    <description>The video protocol configuration</description>
+  </param>
+  <param name="codec" type="Common.VideoStreamingCodec" mandatory="false">
+    <description>The video codec configuration</description>
+  </param>
```

Comment:
The reason these two are separated is just because HMI includes parameters' names in negative SetVideoConfig response. If we use "VideoStreamFormat", we need to clarify the API spec that HMI should use individual "codec" and/or "protocol" names for rejected parameter's list, not "format".

I'm OK to change the spec as you suggested, but I wonder if we can modify the API spec without updating evolution proposal and notifying SDLC members.